guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -71,6 +89,32 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
 
         };
 
+        /**
+         * Allows Clipboard Editor to begin displaying clipboard data, reads the
+         * clipboard data and updates the content of the clipboard editor to the
+         * given data.
+         */
+        var displayAndUpdateClipboardEditor = function displayAndUpdateClipboardEditor() { 
+            isActive = true;
+            element.value = ''
+            element.classList.remove("inactive");
```

Comment:
You can do this same thing with `ng-class` in the template - no need to put this kind of functionality directly in the controller.

Reply:
Or alternatively, you could consider rendering completely different elements based on `isActive` using e.g. `ng-view="isActive"` or `ng-view="!isActive"`. 

That might end up being easier and probably wouldn't require dynamically fiddling CSS classes.